src/System.Private.ServiceModel/src/System/ServiceModel/Diagnostics/SecurityTraceRecordHelper.cs

```diff
@@ -40,5 +40,9 @@ internal static void TraceIdentityDeterminationSuccess(EndpointAddress epr, Endp
         internal static void TraceIdentityDeterminationFailure(EndpointAddress epr, Type identityVerifier)
         {
         }
+
+        internal static void TraceSpnToSidMappingFailure(string spn, Exception e)
+        {
+        }
```

Comment:
Do we have any intent to implement these (currently) empty method bodies?  If so, we should open an issue.  If not, I wonder if we should just nuke these methods and calls.
P.S. I'm asking partly because I'm encountering them in the WS* stuff too and not sure which is the correct choice.